### DISCUSSION TITLE
How to set up SFTP connection?

### DISCUSSION THREAD

**Discussion by dubaaron** [Acode-Foundation/Acode#771] [General]
Hi, 

It looks like SFTP is a feature from the readme, app description, and issues I see, but I can't seem to find in the interface or any documentation how to set up this feature.

Thanks for making this app available to us.

Thank you and Namaste 🙏

aaron

**Comment by sebastianjnuwu** [comment]
> Hi,
> 
> It looks like SFTP is a feature from the readme, app description, and issues I see, but I can't seem to find in the interface or any documentation how to set up this feature.
> 
> Thanks for making this app available to us.
> 
> Thank you and Namaste 🙏
> 
> aaron

Hello everything is fine? To configure the sftp connection you need the `ip` of the machine, the `username` and the password or a Key to make the connection. I'm not sure if that was really the question!

**Reply by dubaaron** [reply]
Hi,

Thanks for your reply. 

I'm asking where in the app do I go to set up an SFTP connection? I can't seem to find the option or interface to do it ...

Thank you --

**Reply by sebastianjnuwu** [reply]
![Screenshot_20230725-131544](https://github.com/deadlyjack/Acode/assets/101356556/48bc82e5-d7be-431a-a642-e2937df0b06b)
![Screenshot_20230725-131547](https://github.com/deadlyjack/Acode/assets/101356556/48347d57-4f51-4533-bc5e-5aeccb308057)
![Screenshot_20230725-131539](https://github.com/deadlyjack/Acode/assets/101356556/1b2f04bf-f368-4fd3-9d0f-8eb1c627a919)
![Screenshot_20230725-131541](https://github.com/deadlyjack/Acode/assets/101356556/988de003-4d1a-437a-85f1-e359963d1bd0)

**Reply by dubaaron** [reply]
Oh, thank you. Found it now. I feel a little stupid, I had not tried that. Since there was a "new file", "save file", etc, I was looking for either a "open" or a "connection" option, didn't think to try just the "files" option. And didn't see the "open folder" option that was on the left menu.

Thanks again, I've got it working now.

**Comment by davegoopot** [comment]
I'm not seeing the "add ..." option in the latest version. Is the functionality still there?

**Reply by UnschooledGamer** [reply]
@davegoopot Hey There, It's still Available as Shown in the previous Thread by sebastianjnuwu.

**Reply by davegoopot** [reply]
![Screenshot_20251113_172133_Acode.jpg](https://github.com/user-attachments/assets/96e57028-4232-44ee-9103-ff7d16ecc554)

Found it! 

Sorry to disturb!

**Reply by UnschooledGamer** [reply]
> ![Screenshot_20251113_172133_Acode.jpg](https://github.com/user-attachments/assets/96e57028-4232-44ee-9103-ff7d16ecc554)
> 
> Found it! 
> 
> Sorry to disturb!

Glad you did. No problem.

Have a Great Day. 
Thanks,
UG.
